scipy/special/tests/test_owens_t.py

```diff
@@ -28,8 +28,15 @@ def test_nans():
 
 
 def test_infs():
+    h, a = 0, np.inf
+    # T(0, a) = 1/2π * arctan(a)
+    res = 1/(2*np.pi) * np.arctan(a)
+    assert_allclose(sc.owens_t(h, a), res, rtol=5e-14)
+    assert_allclose(sc.owens_t(h, -a), -res, rtol=5e-14)
+
     h = 1
-    res = 0.5*sc.erfc(h/np.sqrt(2))
+    # Value approximated through Numerical Integration
```

Comment:
Can you also mention what you integrated? Did you use `scipy.integrate.quad` for numerical integration?

Reply:
Yes, I used scipy.integrate.quad. I computed the value of the T function integral form for the given 'h' and 'a' values. I will mention the resulting expression

Reply:
I have added the integrand function and the reference. Please review it.

Reply:
The comment is very helpful, thanks. I verified the integral from Wikipedia and it looks correct.